omod/src/main/webapp/resources/scripts/visit/filters.js

```diff
@@ -479,6 +479,10 @@ angular.module("filters", [ "uicommons.filters", "constants", "encounterTypeConf
                 });
             };
 
+            var hasMemberThatEvaluatesTrue = function(list, func) {
+              return list.some(func);
+            };
```

Comment:
:upside_down_face:

Reply:
If for some structural reason `hasMemberThatEvaluatesTrue` actually needs to exist, can we just call it `some`?